app/lib/backend_version.rb

```diff
@@ -20,6 +20,10 @@ def oidc_usable?(_)
       false
     end
 
+    def oauth_usable?(service)
```

Comment:
We should put a rolling update here so SaaS users should be able to still do it in API

Reply:
/cc @mayorova do you know when all SaaS users are migrated?

Reply:
@hallelujah no, and I don't expect it will be too soon.